lib/autoload/course/assessment/programming_package.rb

```diff
@@ -117,6 +115,31 @@ def submission_files=(files)
     end
   end
 
+  # Remove the contents of the solution folder so students can't do sneaky things
+  # like generating a report from the solution folder.
+  def remove_solution_files
+    remove_folder_files(SOLUTION_PATH)
+  end
+
+  # Gets the contents of all solution files.
+  #
+  # @return [Hash<Pathname, String>] A hash mapping the file path to the file contents of each
+  #   file.
+  def solution_files
```

Comment:
This is to be consistent with the existing function `submission_files`. There's also `submission_files=` for assignment, so renaming that to `get_submission_files` wouldn't be nice too.